src/Tickets/Flexible_Tickets/Repositories/Event_Repository.php

```diff
@@ -93,14 +93,12 @@ private function include_series_in_meta_value_compare( string $alias ): string {
 
 		// Include Events that are either directly related to a Ticket (Single and Series Tickets) or
 		// are related to a Series that is related to a Ticket (Series Passes).
-		return $wpdb->prepare( "
+		return "
```

Comment:
This is a bit of a nit-pick, but for a multi-line string I think it would be better to use the [heredoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) syntax. Interpolated variables are still allowed in a heredoc string.